resources/user-agents/browsers/iridium/iridium.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Iridium Browser #MAJORVER#.#MINORVER#",
-  "versions": [44, "43.4"],
```

Comment:
All of the patterns in this file only have `#MAJORVER#` in the pattern, so it doesn't make sense to have a minor version here.

This was passing the test (which I've changed) since there are no other "43.x" versions here, but it would end up treating all "43.x" string as "43.4".